settings/language-vala.cson

```diff
@@ -10,3 +10,4 @@
       'stdout'
       'stderr'
     ]
+    'commentStart': '// '
```

Comment:
This was a mistake of mine. It shouldn't be in the pull request. I'll remove it.